src/requirements/data/majors/orie.ts

```diff
@@ -1,16 +1,25 @@
-import { CollegeOrMajorRequirement } from '../../types';
-import { includesWithSingleRequirement, includesWithSubRequirements } from '../checkers-common';
+import { CollegeOrMajorRequirement, Course } from '../../types';
+import { ifCodeMatch, includesWithSubRequirements } from '../checkers-common';
 
 const orieRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Engineering Distribution Courses',
+    description:
+      'ENGRD 2700 and ENGRD 2110 is recommended as the second Engineering Distribution course. ',
+    source: 'https://www.orie.cornell.edu/orie/programs/undergraduate-programs/degree-requirements',
+    checker: includesWithSubRequirements(['ENGRD 2700']),
```

Comment:
Why do we not add ENGRD 2110 here?

Reply:
it's under college & only recommended, not required. eventually we should remove it from college and just add to major